src/vs/workbench/contrib/notebook/browser/diff/diffComponents.ts

```diff
@@ -103,9 +103,9 @@ class PropertyHeader extends Disposable {
 		readonly notebookEditor: INotebookTextDiffEditor,
 		readonly accessor: {
 			updateInfoRendering: (renderOutput: boolean) => void;
-			checkIfModified: (cell: DiffElementCellViewModelBase) => false | { reason: string | undefined };
-			getFoldingState: (cell: DiffElementCellViewModelBase) => PropertyFoldingState;
-			updateFoldingState: (cell: DiffElementCellViewModelBase, newState: PropertyFoldingState) => void;
+			checkIfModified: () => false | { reason: string | undefined };
```

Comment:
@rebornix 
I need to remove this, because we do not have cells and the like for Notebook Metadata.
& I want to re-use this PropertyHeader class